spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/JobExecutionController.java

```diff
@@ -81,6 +81,7 @@ public JobExecutionController(TaskJobService taskJobService) {
 	 * @param pageable  page-able collection of {@code TaskJobExecution}s.
 	 * @param assembler for the {@link TaskJobExecution}s
 	 * @return a list of Task/Job executions
+	 * @throws NoSuchJobExecutionException if
```

Comment:
I see this will be done as part of https://github.com/spring-cloud/spring-cloud-dataflow/issues/1343. Removing this for now.